main.go

```diff
@@ -617,6 +623,11 @@ func main() {
 		log.Warnf("Failed to save the Raw Output, error: %s", err)
 	}
 
+	// Cache swift PM
+	if err := cache.CollectPackagesCache(buildParams.ProjectPath); err != nil {
```

Comment:
why not the `absProjectPath` ?

Reply:
In this case buildParams.ProjectPath is asssigned the value of absProjectPath, but it could be used here indeed too.